lib/auth/windows/windows.go

```diff
@@ -145,6 +166,10 @@ type GenerateCredentialsRequest struct {
 	// CAType is the certificate authority type used to generate the certificate.
 	// This is used to proper generate the CRL LDAP path.
 	CAType types.CertAuthType
+	// CreateUser specifies if Windows user should be created if missing
+	CreateUser bool
+	// Groups are groups that user should be member of
+	Groups []string
```

Comment:
Should this be called `WindowsGroups`?

Reply:
I'm on the fence here, it lives in `windows.go` file and `GenerateCredentialsRequest` is explicitly described as generating windows certs. I'm not sure if we need to mention it again here